### DISCUSSION TITLE
[Bug]: Hyper parameter tuning of machine learning model.

### DISCUSSION THREAD

**Discussion by YashodharDodabhangi** [mindsdb/mindsdb#2759] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Is **hyper parameter tuning** of data while training the classical  machine learning feature is available? for ex: L1 or L2 penalty in logistic regression.

**Comment by paxcema** [comment]
Hi @YashodharDodabhangi, 

It is possible to do this with our AutoML solution via the [USING statement](https://docs.mindsdb.com/sql/create/predictor/#using-statement). Available hyperparameters that you can choose and tune may be found in the [Lightwood documentation pages](https://lightwood.io).
